### ISSUE TITLE
question about the  superpixel segmentation

### ISSUE THREAD

**Issue by colorfuloranges** [danfenghong/ECCV2018_J-Play#2]
Thanks for your sharing! It's very helpful to my research. I have a question about the code. The JPSA is a two-stream multilayered regression model involving the two input sources: 1) pixelwise and 2) superpixelwise spectral signatures and the same output(ground truth). But l do not see the input of the supempixel segmentation part and the superpixel segmentation of images in the code. l dont know if l understand the meaning of the paper correctly?Looking forward to your reply, thank you！

**Comment by danfenghong** [comment]
The code is its previous version (pixel-wise version, i.e., ECCV2018)

colorfuloranges ***@***.***> 于2022年8月1日周一 16:14写道：

**Comment by colorfuloranges** [comment]
Thank you very much for your reply.  If I want to reproduce this algorithm, can the average spectral features extracted after superpixel be directly concatenated after pixel-level spectral features for input?

**Comment by danfenghong** [comment]
yes, your idea is promising and working well, I think. You can try it, if
it works well, please also let me know at your convenience!
Thanks!
Danfeng

colorfuloranges ***@***.***> 于2022年8月1日周一 19:40写道：

**Comment by colorfuloranges** [comment]
Thank you for your encouragement. I'll try it. Thank you very much!
